api/src/main/java/org/hawkular/apm/api/model/analytics/TransactionInfo.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.apm.api.model.analytics;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * This class represents information related to a business transaction associated with
+ * trace fragments.
+ *
+ * @author gbrown
+ */
+public class TransactionInfo {
```

Comment:
Possibly, although sometimes this concept is also known as "User defined transactions", so just trying to keep generic. But may need to consider also changing in other places.